aidants_connect_web/static/scss/attestation-translation.scss

```
@@ -51,52 +52,75 @@ p + p, ul + p {
 
 
 .navbar__home, .container > h1.text-center {
-    /* Hide the header and title on old templates */
-    display: none;
+  /* Hide the header and title on old templates */
+  display: none;
 }
 
 .container {
-    border: none;
+  border: none;
 }
 
 .mandate-translation-row {
-    display: flex;
-    flex-direction: row;
+  display: flex;
+  flex-direction: row;
 
-    .mandate-translation-column {
-        flex: 1 1 0;
-        display: flex;
-    }
+  .mandate-translation-column {
+    flex: 1 1 0;
+    display: flex;
+  }
 }
 
 header.mandate-translation-row .mandate-translation-column:last-child {
-    justify-content: right;
+  justify-content: right;
 }
 
 .subtitle-container {
-    display: flex;
-    flex-direction: row;
-    margin-bottom: 4rem;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 4rem;
 }
 
 .print-button-container .shadowed {
-    max-width: 23rem;
-    float: right;
+  max-width: 23rem;
+  float: right;
 
-    p {
-        margin-bottom: 1rem;
-    }
+  p {
+    margin-bottom: 1rem;
+  }
 
-    button {
-        float: right;
-    }
+  button {
+    float: right;
+  }
 }
 
 ul, ol {
-    /* Cancel padding setting from main-habilitation.scss */
-    padding: revert;
+  /* Cancel padding setting from main-habilitation.scss */
+  padding: revert;
 }
 
 .mandate-translation-other {
-    border-left: 1px solid colors.$blue-aidants;
+  border-left: 1px solid colors.$blue-aidants;
+}
+
+.empty-translation-container {
+  color: colors.$red-aidants;
+
+  & * {
```

Tu voudrais pas ajouter un petit chevron pour ne sélectionner que les descendants de premier niveau ? Comme ça si un jour on a un span dans un div de texte on n'aura pas la surprise de se retrouver avec des trucs inattendus...

```suggestion
  & > * {
```